pkg/mocks/store.go

```diff
@@ -18,11 +18,12 @@ type MockOperationStore struct {
 	sync.RWMutex
 	operations map[string][]batch.Operation
 	Err        error
+	Validate   bool
```

Comment:
@sandrask why isn't "validation" simply the default? By that I mean, why should we have to "enable" this validation? This validation is the default expectation of any sidetree implementation.